services/neutrino/neutrino.js

```diff
@@ -449,12 +497,22 @@ class Neutrino extends EventEmitter {
     else {
       cfilter = this.getCfilterIncrement(line)
       if (cfilter) {
-        this.setLndCfilterHeight(cfilter)
+        this.setNeutrinoCfilterHeight(cfilter)
       }
     }
+  }
 
-    if (height || cfilter) {
-      this.setState(NEUTRINO_CHAIN_SYNC_IN_PROGRESS)
+  /**
+   * Update state if log line indicates that progress has been made in the recovery process.
+   * @param  {String} line log output line
+   */
+  notifyOnRecoveryProgress(line) {
+    let height
+
+    // Check the log line to see if we can parse the current block header height from it.
+    height = this.getRecoveryHeightIncrement(line)
```

Comment:
These two lines can be merged into one const looks like

Reply:
updated